### PR TITLE
JSON module

### PR DESCRIPTION
**Pull request by yrajas**
Add support for JSON.SET and JSON.GET commands using modules.

### CODE REVIEW

File: playground/GarnetJSON/JsonObject.cs [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+        public void Set(string path, string value)
+        {
+            if (path == null)
+                throw new ArgumentNullException(nameof(path));
+
+            if (value == null)
+                throw new ArgumentNullException(nameof(value));
+
+            foreach (var token in jObject.SelectTokens(path).ToList())
```
Comment (Meir017) [inline playground/GarnetJSON/JsonObject.cs:69]:
```suggestion
            foreach (var token in jObject.SelectTokens(path))
```

File: playground/GarnetJSON/JsonObject.cs [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+            : base(obj)
+        {
+            jObject = obj.jObject;
+        }
+
+        public override CustomObjectBase CloneObject() => new JsonObject(this);
+
+        public override void SerializeObject(BinaryWriter writer)
+        {
+            writer.Write(JsonConvert.SerializeObject(jObject));
```
[the remark this answers is not in the document]
Comment (yrajas) [reply playground/GarnetJSON/JsonObject.cs]: Using Newtonsoft primarily to support Jsonpath syntax. So, keeping the APIs uniform for now.
Once System.Text.Json adds support for Jsonpath all of these APIs could be switched over.

[the remark this answers is not in the document]
Comment (Meir017) [reply playground/GarnetJSON/JsonObject.cs]: Pending on https://github.com/dotnet/runtime/issues/31068
Maybe we could use https://github.com/json-everything/json-everything/tree/master/src/JsonPath

[the remark this answers is not in the document]
Comment (badrishc) [reply playground/GarnetJSON/JsonObject.cs]: How well supported is this one versus Newtonsodt which is very popular?

**Pull request merged**